resources/providers/awslib/iam/test_data.go

```diff
@@ -19,20 +19,13 @@ package iam
 
 import (
 	"github.com/aws/aws-sdk-go-v2/aws"
+	iamsdk "github.com/aws/aws-sdk-go-v2/service/iam"
 	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/aws/smithy-go/middleware"
 	"time"
 )
 
 var (
-	keyMetadata = []types.AccessKeyMetadata{
-		{
-			AccessKeyId: aws.String("test_access_key"),
-			CreateDate:  nil,
-			Status:      types.StatusTypeActive,
-			UserName:    aws.String("username"),
-		},
-	}
-
 	virtualMfaDevices = []types.MFADevice{
```

Comment:
Since User properties (name and arn) were changed please check might be also relevant to MFADevice definition.